pkgdown/_pkgdown.yml

```diff
@@ -59,4 +59,8 @@ reference:
       - starts_with("median")
       - starts_with("modal")
       - starts_with("sample")
+  - title: "Data"
+    desc: "These functions resolve objects from the `messydt` class into a single `Date` or other class:"
```

Comment:
Please do not forget to run `pkgdown::build_site()` before we merge the PR so that we know the website looks as expected. Thank you!